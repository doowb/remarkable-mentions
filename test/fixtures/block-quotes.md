> Thanks to @jonchlinkert for helping with the implementation.
