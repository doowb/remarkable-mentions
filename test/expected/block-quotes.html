<blockquote>
<p>Thanks to <a href="https://github.com/jonchlinkert">@jonchlinkert</a> for helping with the implementation.</p>
</blockquote>
